### PR TITLE
Add User Login component

### LINKED ISSUE #5: Implement the user login view
### Description
As part of this issue, we need to create the user login view, which should closely match the provided mockup. The user login will provide the users with a way to log in to the application.

### Tasks

1. Set up the basic React components as required.
2. Implement the layout and design of the homepage, following the provided mockup.
3. Integrate the necessary components and assets to achieve the desired functionality and visual appearance.
4. Ensure responsiveness and compatibility across different devices and screen sizes.

Mockup: https://www.figma.com/file/nehm2ff8aR7NkPerg39fK9/ScholarX-V2-%7C-Wireframes?type=design&node-id=2469-4702&mode=design&t=OFxJkHITzop09S2b-4

image

Note: You may want to mock the API refer to the [API doc](https://www.figma.com/file/nehm2ff8aR7NkPerg39fK9/ScholarX-V2-%7C-Wireframes?type=design&node-id=2469-4702&mode=design&t=OFxJkHITzop09S2b-4) to see the relevant details. 

### Acceptance Criteria

- The login view closely resembles the provided mockup.
- All required sections, elements, and content are present and displayed correctly.
- The login view is responsive and works well on various devices and screen sizes.
- The implementation follows best practices and coding standards.
- Unit tests are written

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to fix #5 

## Goals
- Add User Login component to the repository

## Screenshots
image

##  Checklist
- [ ] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by RandilaP** [comment]
@anjula-sack @kumuditha-udayanga @Piumal1999 Could you please review this

**Comment by anjula-sack** [comment]
Good job! @RandilaP can you resolve the conflicts?

**Comment by anjula-sack** [comment]
Any updates @RandilaP?

**Comment by RandilaP** [comment]
@anjula-sack @kumuditha-udayanga @Piumal1999 @jayasanka-sack Could you please review this

**Comment by RandilaP** [comment]
@anjula-sack @kumuditha-udayanga Could you please review this

File: src/components/UserLogin/UserLogin.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const meta: Meta<typeof UserLogin> = {
+  title: 'Components/UserLogin',
+  component: UserLogin,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const EmptyList: Story = {};
```
Comment (anjula-sack) [inline src/components/UserLogin/UserLogin.stories.tsx]: @RandilaP You need to write a sample story for this.

Comment (RandilaP) [reply src/components/UserLogin/UserLogin.stories.tsx]: @anjula-sack Can you give me an example

**Comment by anjula-sack** [comment]
This has been done by https://github.com/sef-global/scholarx-frontend/pull/59

**Pull request closed without merging**